appservice/src/intTest/groovy/nu/yona/server/PinResetRequestTest.groovy

```diff
@@ -72,14 +81,33 @@ class PinResetRequestTest extends AbstractAppServiceIntegrationTest
 		appService.deleteUser(richard)
 	}
 
-	def 'Clear pin reset request before verification'()
+	def 'Try to verify pin reset confirmation code before end of delay period'()
```

Comment:
This is a weird scenario, a "Hacking attempt" maybe? You could add that prefix to the test case name.